src/core.js

```diff
@@ -441,7 +441,6 @@ $.extend( $.validator, {
 			if ( errors ) {
 				// add items to error list and map
 				$.extend( this.errorMap, errors );
-				this.errorList = [];
```

Comment:
@Arkni awesome thx. so only thing left is a unit test, lets see If I get something crafted together